src/components/organisms/Slider/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import styled from "@emotion/styled";
+import IconTypography from "../../molecules/IconTypography";
+import theme from "../../../themes";
+
+export interface SliderDataProps {
+  id: number;
+  src: string;
+  text?: string;
+  selected: boolean;
+}
+
+export interface SliderProps {
+  data: SliderDataProps[];
+  selected?: number;
+  onClick?: () => void;
+  getSliderId: (id: number) => void;
+}
+
+const responsive = {
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
```

Comment:
Give breakpoints in theme file

Reply:
These breakpoints are not mui breakpoints. These breakpoints are provided by custome 3rd party library.

Reply:
Better to put them in a separate file, so that they can be reused